waspc/data/Generator/templates/react-app/src/auth/forms/Auth.tsx

```diff
@@ -529,7 +529,7 @@ const VerifyEmailForm = (
 
   async function submitForm() {
     if (!token) {
-      setErrorMessage('The token is missing from the URL. Please check the link you received in your email.')
+      setErrorMessage({ title: 'The token is missing from the URL. Please check the link you received in your email.' })
```

Comment:
Fixes this error:
![image](https://user-images.githubusercontent.com/22752307/232827491-d469e714-39b8-496a-8b73-5a6c674471ed.png)